website_docs/markdown/getting-started.md

```diff
@@ -0,0 +1,356 @@
+# Getting Started with OpenTelemetry Python
+
+This guide walks you through instrumenting a Python application with `opentelemetry-python`.
+
+For more elaborate examples, see [examples](https://github.com/open-telemetry/opentelemetry-python/tree/main/docs/examples/).
+
+## Hello world: emit a trace to your console
+
+To get started, install both the opentelemetry API and SDK:
+
+```
+pip install opentelemetry-api
+pip install opentelemetry-sdk
+```
+
+The API package provides the interfaces required by the application owner, as well
+as some helper logic to load implementations.
+
+The SDK provides an implementation of those interfaces. The implementation is designed to be generic and extensible enough
+that in many situations, the SDK is sufficient.
+
+Once installed, you can use the packages to emit spans from your application. A span
+represents an action within your application that you want to instrument, such as an HTTP request
+or a database call. Once instrumented, you can extract helpful information such as
+how long the action took. You can also add arbitrary attributes to the span that provide more insight for debugging.
+
+The following example script emits a trace containing three named spans: “foo”, “bar”, and “baz”:
+
+```
+# tracing.py
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleSpanProcessor,
+)
+
+trace.set_tracer_provider(TracerProvider())
+trace.get_tracer_provider().add_span_processor(
```

Comment:
Is there a reason we use this pattern in examples instead of 

```python
provider = TracerProvider()
provider.add_span_processor(processor)
trace.set_trace_provider(provider)
```

Reply:
saving a variable or two? no specific reason i can think of.

Reply:
Variables make the code harder to understand because it forces the reader to go back a few or many lines before to find the piece of information needed to understand the code. This disadvantage is justified when the variable is used more than once, saving code repetition. Single-use variables have all the problems and none of the advantages of variables. @codeboten approach is right IMO.